configs/textrecog/seg/seg_r31_1by16_fpnocr_toy_dataset.py

```diff
@@ -3,6 +3,7 @@
     '../../_base_/recog_datasets/seg_toy_data.py',
     '../../_base_/recog_models/seg.py',
     '../../_base_/recog_pipelines/seg_pipeline.py',
+    '../../_base_/recog_datasets/academic_test.py'
```

Comment:
The PR was raised to fix the missed schedule in this config file, but this line only add a test dataset config.